wandb/apis/public.py

```diff
@@ -4356,7 +4356,7 @@ def _configure_launch_project_repo(self, launch_project):
         git_info = self._job_info.get("source", {}).get("git", {})
         _fetch_git_repo(
             launch_project.project_dir,
-            git_info["remote"],
+            "git@github.com:wandb/launch-jobs.git",  # git_info["remote"],
```

Comment:
I think you might not have meant to leave this in?

Reply:
Yep, that was for testing, didn't mean to commit it